01 Operator Overloading/OperatorOverloading.Model/InvalidCurrencyException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OperatorOverloading.Model
+{
+    [Serializable]
+    public class InvalidCurrencyException : Exception
```

Comment:
just adding the Serializable attribute will not make it serializable. You need to implement the proper constructor.